### PR TITLE
Merge 6.5 -> 7.0

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Part 2 of 3 of the 3rd bullet of https://github.com/guzzle/guzzle/pull/2712.

### CODE REVIEW

**Comment by Nyholm** [comment]
Could you tell me why this is needed?

**Comment by GrahamCampbell** [comment]
Because the tests fail with the new upcoming psr7 package, without this.

**Comment by Nyholm** [comment]
Sorry. I still don’t quite understand. 

Are you saying tests on guzzle master fails with guzzke/psr master

**Comment by GrahamCampbell** [comment]
Yes, with psr 1.x-dev.

**Comment by Nyholm** [comment]
I don't think we should make more 7.0.x releases. Next release will be 7.1.0. 

I've added https://github.com/guzzle/guzzle/pull/2763 as a test with guzzle/psr7.

File: tests/Exception/RequestExceptionTest.php [hunk 1]
```diff
@@ -130,21 +130,6 @@ public function testExceptionMessageIgnoresEmptyBody()
         self::assertStringEndsWith('response', $e->getMessage());
     }
 
-    public function testCreatesExceptionWithoutPrintableBody()
```
Comment (Nyholm) [inline tests/Exception/RequestExceptionTest.php:133]: I've added an issue for this https://github.com/guzzle/guzzle/issues/2765

File: .github/workflows/static.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: PHPStan
-        uses: OskarStark/phpstan-ga@0.12.28
+        uses: OskarStark/phpstan-ga@0.12.41
```
Comment (Nyholm) [inline .github/workflows/static.yml:19]: I dont mind this, but as you can see in https://github.com/OskarStark/phpstan-ga/issues/31, we are already running 0.12.42 =)

**Review by Nyholm — CHANGES_REQUESTED** [review]
I think this PR should be closed. 

We could consider use parts of it to fix https://github.com/guzzle/guzzle/issues/2765 but Im sure there are better solutions than just remove the test..

Also, we should not target 7.0 branch.

File: tests/Exception/RequestExceptionTest.php (hunk 1 above)
Comment (GrahamCampbell) [reply tests/Exception/RequestExceptionTest.php:133]: We already discussed this a few months ago and decided this was not a bug.

File: .github/workflows/static.yml (hunk 2 above)
Comment (GrahamCampbell) [reply .github/workflows/static.yml:19]: 0.12.42 isn't available on docker hub.

Comment (GrahamCampbell) [reply .github/workflows/static.yml:19]: So I couldn't use it to generate the baseline locally.

**Comment by GrahamCampbell** [comment]
I think we should just merge this, then I can do a 7.0 -> 7.1 merge, after.

**Comment by GrahamCampbell** [comment]
Doesn't matter if there will not be another 7.0.x release. Indeed, there are already other commits on 7.0 that will never be tagged.

**Comment by GrahamCampbell** [comment]
Will just target master then, and skip the 7.0 merge, since that seems to be what you want. :)

**Comment by GrahamCampbell** [comment]
Makes little difference, other than the fact the github branch thing will show 6.5 as being ahead of 7.0, but not 7.1.

**Comment by Nyholm** [comment]
If you just make a PR that will fix https://github.com/guzzle/guzzle/issues/2765, then I would be super happy =)

**Pull request closed without merging**